it/content/src/main/content/jcr_root/conf/global/settings/cloudconfigs/marketo/marketoconfig/.content.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        jcr:createdBy="admin"
+        jcr:title="marketoConfig"
+        client_id="03358092-343f-4fab-8d63-5066419c8122"
+        munchkinId="797-MKX-025"
+        access_token_uri="https://797-MKX-025.mktorest.com/identity/oauth/token"
+        dataSourceType="MARKETO"
+        jcr:lastModifiedBy="admin"
+        jcr:created="Mon Oct 07 2024 08:51:27 GMT+0530"
+        cq:lastModified="Mon Oct 07 2024 08:51:27 GMT+0530"
+        name="marketoconfig"
+        sling:resourceSuperType="cq/cloudserviceconfigs/components/configpage"
+        marketoBaseUrl="797-MKX-025.mktorest.com"
```

Comment:
Since we have pasted the entire marketo connector configuration, why aren't we verifying the actual runtime use-case, like submit to marketo

Reply:
changed this. We are in process of acquiring an actual account for testing, even for load testing also.
Once that is available we can use that. This account we have is personal and can rely on this.